scalameta/tests/jvm/src/test/resources/semanticdb.expect

```diff
@@ -16,14 +16,30 @@ Names:
 Symbols:
 _root_.example. => package example
 _root_.example.Example. => final object Example
-_root_.example.Example.main([Ljava/lang/String;)V. => def main: (args: Array[String])Unit
-_root_.example.Example.main([Ljava/lang/String;)V.(args) => param args: scala.Array[String]
+_root_.example.Example.main([Ljava/lang/String;)V. => def main: (args: scala.Array[scala.Predef.String])scala.Unit
+  [46..50): Unit => _root_.scala.Unit#
+  [13..18): Array => _root_.scala.Array#
+  [19..24): scala => _root_.scala.
+  [25..31): Predef => _root_.scala.Predef.
+  [32..38): String => _root_.scala.Predef.String#
```

Comment:
Yes, I intentionally left out for example the symbols of parameter names since I'm not convinced they're useful. We can add more resolved names later after demand.